analysis/src/CompletionFrontEnd.ml

```diff
@@ -149,11 +150,19 @@ let findNamedArgCompletable ~(args : arg list) ~endPos ~posBeforeCursor
   let rec loop args =
     match args with
     | {label = Some labelled; exp} :: rest ->
+      (* Figure out if we're completing the labelled argument name, or assigning to the labelled argument *)
       if
         labelled.posStart <= posBeforeCursor
         && posBeforeCursor < labelled.posEnd
+        && charBeforeCursor <> Some '='
+        (* We need to account for empty assignments being parsed as regular labelled arguments without an assignment *)
       then Some (Completable.CnamedArg (contextPath, labelled.name, allNames))
-      else if exp.pexp_loc |> Loc.hasPos ~pos:posBeforeCursor then None
+      else if
+        exp.pexp_loc |> Loc.hasPos ~pos:posBeforeCursor
+        || charBeforeCursor = Some '='
```

Comment:
Let's go slowly and stick to the script.
This is what you do when there is no other choice.
I would like to see how far we can go before trying to do this.

Also, we don't know if it's the character before the cursor.